cupy/core/_scalar.pxd

```diff
@@ -1,35 +1,11 @@
 from libc.stdint cimport int8_t
-from libc.stdint cimport int16_t
-from libc.stdint cimport int32_t
-from libc.stdint cimport int64_t
-from libc.stdint cimport uint8_t
-from libc.stdint cimport uint16_t
-from libc.stdint cimport uint32_t
-from libc.stdint cimport uint64_t
 
 from cupy.cuda.function cimport CPointer
 
 
-cdef union Scalar:
-    bint bool_
-    int8_t int8_
-    int16_t int16_
-    int32_t int32_
-    int64_t int64_
-    uint8_t uint8_
-    uint16_t uint16_
-    uint32_t uint32_
-    uint64_t uint64_
-    float float32_
-    double float64_
-    float complex complex64_
-    double complex complex128_
```

Comment:
Note for later reference: These two `complex` lines are causes of this issue; unrestricted unions are implemented in GCC 4.6 whereas CentOS 6 only provides GCC 4.4.